package.json

```diff
@@ -23,6 +23,7 @@
 		"plugin-zip": "wp-scripts plugin-zip",
 		"start": "wp-scripts start",
 		"test:e2e": "wp-scripts test-e2e",
+		"tests:e2e-all": "playwright test --config=playwright.config.js",
```

Comment:
@prashantabellad as discussed during the meeting today. Let's consolidate lines 25/26. cc @mauteri